From 1343959187532c45d83bb5902182e2686ce24d69 Mon Sep 17 00:00:00 2001
From: Thierry Reding <thierry.reding@avionic-design.de>
Date: Wed, 25 Jul 2012 16:24:49 +0200
Subject: [PATCH 127/429] iommu: Include linux/types.h

commit 76582d0a68ab6c46987dd678165c26ad81c44cdb upstream

The linux/iommu.h header uses types defined in linux/types.h but doesn't
include it.

Signed-off-by: Thierry Reding <thierry.reding@avionic-design.de>
Signed-off-by: Joerg Roedel <joerg.roedel@amd.com>
Signed-off-by: Tiejun Chen <tiejun.chen@windriver.com>
---
 include/linux/iommu.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/linux/iommu.h b/include/linux/iommu.h
index 23caf92..e0f2183 100644
--- a/include/linux/iommu.h
+++ b/include/linux/iommu.h
@@ -20,6 +20,7 @@
 #define __LINUX_IOMMU_H
 
 #include <linux/errno.h>
+#include <linux/types.h>
 
 #define IOMMU_READ	(1)
 #define IOMMU_WRITE	(2)
-- 
1.7.5.4

